audio_service/android/src/main/java/com/ryanheise/audioservice/AudioService.java

```diff
@@ -286,6 +286,7 @@ public int onStartCommand(final Intent intent, int flags, int startId) {
     }
 
     public void stop() {
+        notificationCreated = false;
```

Comment:
Just wondering whether it would be best to consider `androidResumeOnClick` here, which is currently used to pass in the argument to `stopForeground`.